src/components/LoginUser.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+        <b-form @submit="onSubmit">
+          <b-form-group>
+            <b-form-input v-model="username" type="text" placeholder="Username" required/>
+          </b-form-group>
+          <b-form-group :invalid-feedback="invalidFeedback">
+            <b-form-input v-model="password" type="password" placeholder="Password" required/>
```

Comment:
If you weren't aware, 'v-model' is awesome with computed properties. You can link them directly, and change your vuex state smoothly without having to worry about methods.